migrations/versions/20210808_121713_2e6a53303c9b_.py

```diff
@@ -1,16 +1,16 @@
 """empty message
 
-Revision ID: dd60e491f79e
+Revision ID: 2e6a53303c9b
```

Comment:
i had to throw the migration from before out, since it generated some broken sql when setting the table "unlogged"